### PR TITLE
[W4.11e][W16-1] Shanna Lim

### PR DESCRIPTION
**Pull request by kengwoon**
Ready for review

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -36,6 +36,25 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNullTest() throws Exception {
```
Comment (bqnguyen94) [inline test/java/seedu/addressbook/common/UtilsTest.java:40]: You can add more test cases, such as empty `Objects`, or `Objects` that contain other data types like numbers, arrays, etc.

**Review by bqnguyen94 — COMMENTED** [review]
Looks good. Please close the PR after reading this.
